shell/browser/native_window_mac.h

```diff
@@ -267,6 +267,8 @@ class NativeWindowMac : public NativeWindow,
   // Maximizable window state; necessary for persistence through redraws.
   bool maximizable_ = true;
 
+  bool user_set_bounds_maximized_ = true;
```

Comment:
Should this be defaulting to false? I'm not sure `SetBounds` is necessarily invoked by the constructor. Could be why the other tests are failing on this PR - defaults to true and doesn't get updated since the `SetBounds` code path isn't taken.

Reply:
yeah it should be!